UPGRADING.md

```diff
@@ -46,6 +50,148 @@ reqSnapshots := &opensearchapi.SnapshotDeleteRequest{
   Snapshot: sliceSnapshotsToDelete,
 ```
 
+## Upgrading to >= 3.0.0
+
+Version 3.0.0 contains a large restructure of the lib braking all existing implementations.
```

Comment:
Typo: breaking

Keep it short?

"Version 3.0 is a major refactor of the client."